### PR TITLE
divideSeriesLists function

### PR DESCRIPTION
**Pull request by leoleovich**
Iterates over a two lists and divides list1[0] by list2[0], list1[1] by list2[1] and so on.

### CODE REVIEW

**Comment by leoleovich** [comment]
I am checking tests

**Comment by DanCech** [comment]
At first glance this looks really good :)

File: webapp/graphite/render/functions.py [hunk 1]
```diff
@@ -616,6 +616,41 @@ def asPercent(requestContext, seriesList, total=None):
 
   return resultList
 
+def divideSeriesLists(requestContext, dividendSeriesList, divisorSeriesList):
+  """
+  Iterates over a two lists and divides list1[0] by list2[0], list1[1] by list2[1] and so on.
+  The lists need to be the same lenght
```
Comment (obfuscurity) [inline webapp/graphite/render/functions.py]: `lenght`

Comment (leoleovich) [inline webapp/graphite/render/functions.py]: Fixed

**Comment by deniszh** [comment]
Thanks, @leoleovich, merged!

**Pull request merged**